lib/main_containers.pm

```diff
@@ -151,8 +151,8 @@ sub load_host_tests_docker {
     load_firewall_test($run_args) unless (is_public_cloud || is_openstack || is_microos);
     unless (is_sle("<=15") && is_aarch64) {
         # these 2 packages are not avaiable for <=15 (aarch64 only)
-        # zypper-docker is not available in factory and in SLE Micro/MicroOS
-        loadtest 'containers/zypper_docker' unless (is_tumbleweed || is_sle_micro || is_microos || is_leap_micro);
+        # zypper-docker is only available on SLES < 15-SP6
+        loadtest 'containers/zypper_docker' if (is_sle("<15-SP6"));
```

Comment:
Shouldn't it stay for older leap and sle-micro as well?

Reply:
Right - https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18774